src/utils/exportUtils/HtmlExport.ts

```diff
@@ -0,0 +1,459 @@
+    text-decoration: underline;
+}
+
+.mx_page_header {
+    position: fixed;
+    z-index: 10;
+    background-color: #ffffff;
+    width: 100%;
+    border-bottom: 1px solid #e3e6e8;
+}
+
+.mx_page_header .mx_content {
+    width: 480px;
+    margin: 0 auto;
+    border-radius: 0 !important;
+}
+
+.mx_page_header a.mx_content {
+    background-repeat: no-repeat;
+    background-position: 24px 21px;
+    background-size: 24px 24px;
+}
+
+.mx_bold {
+    color: #212121;
+    font-weight: 700;
+}
+
+.mx_details {
+    color: #70777b;
+}
+
+.mx_page_header .mx_content .mx_text {
+    padding: 24px 24px 22px 24px;
+    font-size: 22px;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    text-align: center;
+}
+
+.mx_page_header a.mx_content .mx_text {
+    padding: 24px 24px 22px 82px;
+}
+
+.mx_page_body {
+    padding-top: 64px;
+    width: 700px;
+    margin: 0 auto;
+}
+
+.mx_userpic {
+    display: block;
+    border-radius: 50%;
+    overflow: hidden;
+}
+
+.mx_userpic .mx_initials {
+    display: block;
+    color: #fff;
+    text-align: center;
+    text-transform: uppercase;
+    user-select: none;
+}
+
+a.mx_block_link {
+    display: block;
+    text-decoration: none !important;
+    border-radius: 4px;
+}
+
+a.mx_block_link:hover {
+    text-decoration: none !important;
+    background-color: #f5f7f8;
+}
+
+.mx_history {
+    padding: 16px 0;
+}
+
+.mx_message {
+    margin: 0 -10px;
+    transition: background-color 2.0s ease;
+}
+
+div.mx_selected {
+    background-color: rgba(242,246,250,255);
+    transition: background-color 0.5s ease;
+}
+
+.mx_service {
+    padding: 10px 24px;
+}
+
+.mx_service .mx_body {
+    text-align: center;
+}
+
+.mx_message .mx_userpic .mx_initials {
+    font-size: 16px;
+}
+
+.mx_default {
+    padding: 10px;
+}
+
+.mx_default.mx_joined {
+    margin-top: -10px;
+}
+
+.mx_default .mx_from_name {
+    font-weight: 700;
+    padding-bottom: 5px;
+}
+
+.mx_default .mx_body {
+    margin-left: 60px;
+}
+
+.mx_default .mx_text {
+    word-wrap: break-word;
+    line-height: 150%;
+}
+
+.mx_default .mx_reply_to,
+.mx_default .mx_media_wrap {
+    padding-bottom: 5px;
+}
+
+.mx_default .mx_media {
+    margin: 0 -10px;
+    padding: 5px 10px;
+}
+
+.mx_default .mx_media .mx_fill,
+.mx_default .mx_media .mx_thumb {
+    width: 48px;
+    height: 48px;
+    border-radius: 50%;
+}
+
+.mx_default .mx_media .mx_fill {
+    background-repeat: no-repeat;
+    background-position: 12px 12px;
+    background-size: 24px 24px;
+}
+
+.mx_default .mx_media .mx_title {
+    padding-top: 4px;
+    font-size: 14px;
+}
+
+.mx_default .mx_media .mx_description {
+    color: #000000;
+    padding-top: 4px;
+    font-size: 13px;
+}
+
+.mx_default .mx_media .mx_status {
+    padding-top: 4px;
+    font-size: 13px;
+}
+
+.mx_default .mx_photo {
+    display: block;
+}
+`;
+
+
+const userColors = [
+    "#64bf47",
+    "#4f9cd9",
+    "#9884e8",
+    "#fb6238",
+    "#00cdac",
+    "#ff5eaa",
+];
```

Comment:
these colours don't seem to match out theming

Reply:
Will modify them